src/store/RelayStoreData.js

```diff
@@ -86,10 +84,7 @@ class RelayStoreData {
    * Get the data set backing actual Relay operations. Used in GraphQLStore.
    */
   static getDefaultInstance(): RelayStoreData {
-    if (!_instance) {
-      _instance = new RelayStoreData();
-    }
-    return _instance;
+    return RelayStore.getStoreData();
```

Comment:
Thanks for clarifying - that makes total sense, and :+1: to replacing `RelayStoreData.getDefaultInstance()` with `RelayStore.getStoreData()`.